GDevelop.js/__tests__/Core.js

```diff
@@ -1108,6 +1108,29 @@ describe('libGD.js', function () {
       expect(allResources.get(0)).toBe('Used');
       project.delete();
     });
+
+    it('should not remove loading screen image when removing useless resources', function () {
+      var project = gd.ProjectHelper.createNewGDJSProject();
+      var resource1 = new gd.ImageResource();
+      resource1.setName('LoadingScreenImage');
+      project.getLoadingScreen().setBackgroundImageResourceName(resource1.getName())
+
+      expect(project.getLoadingScreen().getBackgroundImageResourceName()).toBe('LoadingScreenImage');
+
+      let worker = new gd.ResourcesInUseHelper();
```

Comment:
Uber Nitpicking: const instead of let (here and below)

(The JS community is raging on this issue, but overall I go for the most specific keyword, `const`, and fallback to `let` when not needed)
(other newer language, I think Rust, have a keyword `let` and `let mut` for mutable stuff 
- validating that you should prefer const as much as possible)